src/components/button/Button.js

```diff
@@ -191,6 +191,14 @@ export default class ButtonComponent extends BaseComponent {
         this.loading = false;
       });
     }
+
+    if (this.component.action === 'event') {
+      this.on('change', (value) => {
+        const isValid = this.root.isValid(value.data, true);
```

Comment:
The validity is passed as ```value.isValid``` so I would prefer to use that over an additional check on root. Should probably say.

```
const isValid = value && value.hasOwnProperty('isValid') ? value.isValid : this.root.isValid(value.data, true);
```

Reply:
Ok, I'll push commit with update soon.